Update index.mdx

meker12 opened the pull request:
## Purpose of this pull request

Update the storefront get started procedure with current workflow using Site Creator Tool.
Other changes:

- Changed references to site import tool to "Site Creator Tool"
- Moved info about Site Importer tool to import existing site to a note.
- Added AEM CLI tool installation as a prerequisite to support aem CLI commands which are referenced in EDS-related documentation
- Updated note about configuration service to point to the AEM configuration service documentation. Instructions are no longer included in the boilerplate readme. Additionally, the AEM workflow is to create a site first, and then set up the Configuration service.

### Associated JIRA ticket

[COMOPT-553](https://jira.corp.adobe.com/browse/COMOPT-553)

## Affected pages

https://experienceleague.adobe.com/developer/commerce/storefront/get-started/#step-by-step

meker12 committed:
Update index.mdx

bdenham committed:
Merge branch 'develop' into eds-storefront-setup-updates

meker12 committed:
Fix link to Storefront configuration reference

meker12 committed:
Merge branch 'develop' into eds-storefront-setup-updates

meker12 committed:
Fix link to storefront configuration reference

meker12 committed:
- Add AEM CLI tool installation to prerequisites
- Add icon for command line tool (aemcli)

meker12 committed:
Update index.mdx

Fix link

sirugh commented on src/content/docs/get-started/index.mdx:
```diff
@@ -224,13 +235,13 @@ The Code Sync app redeploys your storefront site whenever you push or merge chan
-To link your storefront repo to your Commerce backend:
-- Duplicate the [demo-config.json](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/demo-config.json) from your storefront repo and rename it to `config.json`. The `demo-config.json` is only meant as a starting point for your site's configuration.
-- Alternatively, use the [config generator tool](https://da.live/app/adobe-commerce/storefront-tools/tools/config-generator/config-generator) to automatically generate a `config.json` file for your site.
+To link your storefront repo to your commerce backend, you can duplicate the [demo-config.json](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/demo-config.json) from your storefront repo and rename it to `config.json`.  The configuration settings in this file depend on which Commerce backend you are using: Adobe Commerce, Adobe Commerce as a Cloud Service, or Adobe Commerce Optimi ...
```
What if we instead just reference [config-generator](https://da.live/app/adobe-commerce/storefront-tools/tools/config-generator/config-generator) with a note to reference the demo site config if needed: https://www.aemshop.net/config.json

sirugh replied:
That's basically what I had before this change - but you deleted it :)

bdenham committed:
Merge branch 'develop' into eds-storefront-setup-updates

meker12 committed:
Revise linking instructions for commerce backend

meker12 committed:
Update content tool reference in getting started guide

Removed reference to Site Importer tool because the process is for setting up a new store front.

meker12 committed:
Add import instructions for Edge Delivery Service site

Add note about importing an existing site into Document Author.

meker12 committed:
Clarify site import process for Commerce projects

meker12 committed:
Modify note for EDS site import and fix link syntax

Updated note section to reflect importing existing EDS site and corrected link format in the configuration reference.

meker12 committed:
Fix link in get-started documentation

bdenham reviewed (approved):
Approved pending changes to the numbering without using the <Steps> component

sirugh commented on src/content/docs/get-started/index.mdx:
```diff
@@ -224,13 +239,14 @@ The Code Sync app redeploys your storefront site whenever you push or merge chan
-To link your storefront repo to your Commerce backend:
-- Duplicate the [demo-config.json](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/demo-config.json) from your storefront repo and rename it to `config.json`. The `demo-config.json` is only meant as a starting point for your site's configuration.
-- Alternatively, use the [config generator tool](https://da.live/app/adobe-commerce/storefront-tools/tools/config-generator/config-generator) to automatically generate a `config.json` file for your site.
+To link your storefront repo to your commerce backend, use either of the following methods:
```
`either` - I only count one option, unless the diff view here is wonky :)

sirugh commented on src/content/docs/get-started/index.mdx:
```diff
@@ -224,13 +239,14 @@ The Code Sync app redeploys your storefront site whenever you push or merge chan
 
-Read more about Commerce configuration [here](/setup/configuration/commerce-configuration).
+1. Get information about the values required for your project from the [Storefront configuration reference](/setup/configuration/commerce-configuration) to get information about the values required for your project.
+2. Update the `config.json` file with the values for your project.
+3. Commit and push the updated file to the repo.
```
We should not lose reference to "config service". It's the correct way to manage config, IMO. Technically a repo file will work, but I suggest config service publicConfig long term.

meker12 replied:
The info related to the Config Service is still included on line 196.

meker12 replied:
I updated the note. Readme no longer provides info about using Config Service. Additionally, the config service docs start with assumption that you've already created a site using the config.json.  

Updated the note and linked to the Configuration Service documentation and provided more context about when to use it. 

New note:

After you have created your storefront, you can migrate the configuration to use the Configuration Service which supports advanced use cases like repoless configuration and overlays. For more information, see the [Configuration Service](https://www.aem.live/docs/config-service-setup) documentation.

meker12 commented on src/content/docs/get-started/index.mdx:
```diff
@@ -224,13 +239,14 @@ The Code Sync app redeploys your storefront site whenever you push or merge chan
-To link your storefront repo to your Commerce backend:
-- Duplicate the [demo-config.json](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/demo-config.json) from your storefront repo and rename it to `config.json`. The `demo-config.json` is only meant as a starting point for your site's configuration.
-- Alternatively, use the [config generator tool](https://da.live/app/adobe-commerce/storefront-tools/tools/config-generator/config-generator) to automatically generate a `config.json` file for your site.
+To link your storefront repo to your commerce backend, use either of the following methods:
+you can use the [config generator tool](https://da.live/app/adobe-commerce/storefront-tools/tools/config-generator/config-generator) to automatically generate a `config.json` file for your site based on your Commerce backend—Adobe Commerce, Adobe Commerce as a Cloud Service, or Adobe Commerce Optimizer.
```
```suggestion
To link your storefront repo to your commerce backend, you can use the [config generator tool](https://da.live/app/adobe-commerce/storefront-tools/tools/config-generator/config-generator) to automatically generate a `config.json` file for your site based on your Commerce backend—Adobe Commerce, Adobe Commerce as a Cloud Service, or Adobe Commerce Optimizer.
```

meker12 committed:
Update src/content/docs/get-started/index.mdx

meker12 committed:
Update index.mdx

meker12 commented on src/content/docs/get-started/index.mdx:
```diff
@@ -179,8 +194,7 @@ Edge Delivery Services requires site configuration details to know what to rende
 <Aside type="note" title="Config Service Approach">
-  You can onboard instead by using the Config Service via the [Helix Admin API](https://www.aem.live/docs/admin.html). To do this, follow the steps in the [boilerplate readme](https://github.com/hlxsites/aem-boilerplate-commerce?tab=readme-ov-file#initial-setup). See the Configuration Service documentation for more information: https://www.aem.live/docs/config-service-setup
-</Aside>
+After you have created your first storefront, you can migrate the configuration to use the Configuration Service which supports advanced use cases like repoless configuration and overlays. For more information, see the [Configuration Service](https://www.aem.live/docs/config-service-setup) documentation.
```
```suggestion
After you have created your storefront, you can migrate the configuration to use the Configuration Service which supports advanced use cases like repoless configuration and overlays. For more information, see the [Configuration Service](https://www.aem.live/docs/config-service-setup) documentation.
```

meker12 committed:
Update src/content/docs/get-started/index.mdx

meker12 committed:
Correct formatting in Configuration Service note

Fixed formatting issues in the note about the Configuration Service.

meker12 committed:
Merge branch 'develop' into eds-storefront-setup-updates

meker12 commented:
> Approved pending changes to the numbering without using the component

@bdenham  Can you clarify what you mean

meker12 committed:
Fix step number format to use astro component

meker12 commented:
> Approved pending changes to the numbering without using the component

I added the <Steps> component to the procedures that didn't have it.

bdenham committed:
Merge branch 'develop' into eds-storefront-setup-updates
